frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx

```diff
@@ -111,20 +120,20 @@ export const MigrateButton = ({
 
             // update service
             await ServicesService.updateService({
-              stakingProgramId,
+              stakingProgramId: stakingProgramIdToMigrateTo,
               serviceTemplate,
-              serviceUuid: serviceTemplate.service_config_id,
+              serviceUuid: serviceConfigId,
               deploy: true,
               useMechMarketplace:
-                stakingProgramId === StakingProgramId.BetaMechMarketplace,
+                stakingProgramIdToMigrateTo ===
+                StakingProgramId.PearlBetaMechMarketplace,
+              chainId: homeChainId,
             });
 
             // start service after updating
-            await ServicesService.startService(
-              serviceTemplate.service_config_id,
-            );
+            await ServicesService.startService(serviceConfigId);
 
-            await updateStakingProgram(); // TODO: refactor to support single staking program & multi staking programs, this on longer works
+            // await updateStakingProgram(); // TODO: refactor to support single staking program & multi staking programs, this on longer works
```

Comment:
unsure what this should do, would appreciate any advice

Reply:
was previously a function refetch staking program data